main.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Branch names aren't variables so you shouldn't name it firstCommit, use a hyphenated structure like 'initial-commit'.  Also, you can start making better commit messages since your projects are more complicated now, a good way is a short descriptive title, line separator, then short summary.  If its a small commit, you can just have a short title.

Initial commit

Initial implementation of base angular views and controllers, added css styling, set up single page router.